### DISCUSSION TITLE
HammerDb 4.1 installation error on Windows Server 2003 x64

### DISCUSSION THREAD

**Discussion by CCDenis1983** [TPC-Council/HammerDB#258] [Q&A] [answered]
Hi all,
I'm trying to install HammerDB 4.1 on a machine with Windows Server 2003 x64 OS, but during the wizard, the following error appears: **.\bin\wish86t.exe is not a valid Win32 application.**

Thanks in advance for your kindly support.
Denis

![image](https://user-images.githubusercontent.com/86954569/124445772-21f21e00-dd80-11eb-8201-eb4fa318cf70.png)

**Comment by sm-shaw** [comment, marked as answer]
Hi,
HammerDB v4.1 is a 64-bit/x64 application so is not a Win32 application. 
If you browse https://github.com/TPC-Council/HammerDB/releases v2.23 was the last release for 32-bit x86 Windows.
v3.0 onwards have all been 64-bit only.
